app/metrics.yaml

```diff
@@ -382,6 +382,129 @@ search.default_engine:
     - fenix-core@mozilla.com
     expires: "2019-09-01"
 
+bookmarks_management:
+  bookmark_in_new:
```

Comment:
`bookmark_in_new` without the docs is a little ambiguous.

The parent key becomes a part of the event name which allows you to express the events while staying under the key length. So you could do something like

```
bookmark:
  open_in_tab:
  open_in_private_tabs:
```

Reply:
That helps. I'll rename them.